dds/DCPS/ReactorTask.inl

```diff
@@ -0,0 +1,63 @@
+/*
+ *
+ *
+ * Distributed under the OpenDDS License.
+ * See: http://www.opendds.org/license.html
+ */
+
+#include "ace/Reactor.h"
+#include "debug.h"
+
+OPENDDS_BEGIN_VERSIONED_NAMESPACE_DECL
+
+ACE_INLINE ACE_Reactor*
+OpenDDS::DCPS::ReactorTask::get_reactor()
+{
+  if (DCPS_debug_level >= 6) {
```

Comment:
I realize these are here because the previous version had a similar thing for transport logging, but the "levels" aren't 1-to-1.  I'd rather just not have not, I can't see the benefit.  There are debuggers and dynamic trace tools if someone wants to see every function that gets called, that's not what logging is for.